helpers.js

```diff
@@ -901,7 +901,7 @@ function getMiscInfoHtml(item, datetime) {
 
         if (item.Type == "Audio") {
 
-            miscInfo.push(datetime.getDisplayRunningTime(item.RunTimeTicks));
+            miscInfo.push(getDisplayRunningTime(item.RunTimeTicks));
```

Comment:
Does this method exist elsewhere?

Reply:
Good catch, I was a bit too fast in my cleaning frenzy. I'll throw it into helpers.js.